fedmsg/core.py

```diff
@@ -163,6 +164,7 @@ def __init__(self, **config):
             pass
 
         # Cleanup.  See https://bit.ly/SaGeOr for discussion.
+        # arg signature - weakref.ref(object [, callback])
```

Comment:
I just put the argument signature here, in case someone doesn't know about it (l had to look it up for example)